controllers/propagator/propagation.go

```diff
@@ -187,7 +187,8 @@ func (r *PolicyReconciler) cleanUpPolicy(instance *policiesv1.Policy) error {
 		return nil
 	}
 
-	log.V(2).Info("Deleting %d replicated policies because root policy was deleted", len(replicatedPlcList.Items))
+	log.V(2).Info(
+		fmt.Sprintf("Deleting %d replicated policies because root policy was deleted", len(replicatedPlcList.Items)))
```

Comment:
Good catch, though I think it should be key-value pairs like:
```suggestion
		"Deleting replicated policies because root policy was deleted", "count", len(replicatedPlcList.Items))
```